src/modules/0.0.22/aws_rds/entity/parameter_group.ts

```diff
@@ -83,4 +88,13 @@ export class ParameterGroup {
     nullable: true,
   })
   parameters?: Parameter[];
+
+  // This column is joined to `aws_regions` manually via hooks in the `../sql` directory
```

Comment:
Just a minor comment, you can use Alejandro's approach to avoid having `{before,after}_install.sql` scripts.
https://github.com/iasql/iasql-engine/pull/1364/files#diff-610483bfe761aea785477f5c21be25ad1405dabde226aafe32780d230bd71145R133-R141

Reply:
Oh, right! Will make that update.